pages/share-link.js

```diff
@@ -0,0 +1,8 @@
+import ShareLink from "../components/ShareLink/ShareLink";
```

Comment:
As we're using the ShareLink component directly on the product page (pages/products/[id].js), I believe we can delete this (pages/share-link.js) file.